ecs-init/engine/engine.go

```diff
@@ -33,6 +35,11 @@ const (
 	terminalSuccessAgentExitCode = 0
 	terminalFailureAgentExitCode = 5
 	upgradeAgentExitCode         = 42
+	minServiceStartRetryTime     = time.Millisecond * 500
```

Comment:
nit: these constants should be prefixed the same as they're grouped to serve a common function.

```suggestion
	serviceMinStartRetryTime     = time.Millisecond * 500
```

Reply:
Sure, I will give them the same prefix but I'm going to go with `serviceStart`